<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-OJU0P6V" failures="7" tests="43" name="DataExchangeAdmin" time="218.206" errors="0" timestamp="2024-04-25T15:11:22 IST">
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="Credentilas" time="10.453"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="Admin_APIConnectivity" time="3.153"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="AdminDashboard" time="3.825"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="Admin_SearchUsersByName" time="5.725"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="Admin_IsUserPresent" time="1.203"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="CustomAttributes" time="14.843"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="UpdateSystemUser" time="1.336"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="EnableDisableSystemUser" time="1.032"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="AgencyDistribution" time="1.533"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="AgencyAll" time="0.902"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="batch_fileupload" time="0.953">
    <failure type="java.lang.AssertionError" message="status code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: status code is not 200 expected [200] but found [400]
at com.qa.tests.DataExchangeAdmin.batch_fileupload(DataExchangeAdmin.java:657)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- batch_fileupload -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="historicalbatches" time="0.876"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="batch_downloadfile" time="0.86">
    <failure type="java.lang.AssertionError" message="status code is not 200 expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: status code is not 200 expected [200] but found [405]
at com.qa.tests.DataExchangeAdmin.batch_downloadfile(DataExchangeAdmin.java:780)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- batch_downloadfile -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="batchdetail" time="0.898"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="downloadbatchdetail" time="1.368"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="client" time="0.872">
    <failure type="java.lang.AssertionError" message="status code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: status code is not 200 expected [200] but found [400]
at com.qa.tests.DataExchangeAdmin.client(DataExchangeAdmin.java:844)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- client -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="mapping" time="0.87">
    <failure type="java.lang.AssertionError" message="status code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: status code is not 200 expected [200] but found [400]
at com.qa.tests.DataExchangeAdmin.mapping(DataExchangeAdmin.java:875)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- mapping -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="alertmessage" time="0.857"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reference_agency" time="0.885"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reference_role" time="0.881"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reference_userdisablereason" time="0.886"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reeference_template" time="1.476"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_landing" time="0.849"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_failedlogins" time="43.596"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_activelogins" time="23.006">
    <failure type="org.json.JSONException" message="JSONArray[0] not found.">
      <![CDATA[org.json.JSONException: JSONArray[0] not found.
at org.json.JSONArray.get(JSONArray.java:264)
at com.qa.util.TestUtil.getValueByjpath(TestUtil.java:16)
at com.qa.tests.DataExchangeAdmin.reports_activelogins(DataExchangeAdmin.java:1136)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- reports_activelogins -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_loginhistory" time="19.5"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_newaccounts" time="3.185"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_queryhistory" time="4.63"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_seaechesperformedbyuser" time="3.935"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_multiauthorization" time="5.368"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_entitydetails" time="3.137"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="reports_employeeworkinghours" time="27.147"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="searchreason" time="3.376">
    <failure type="org.json.JSONException" message="JSONObject[&amp;quot;clientname&amp;quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["clientname"] not found.
at org.json.JSONObject.get(JSONObject.java:572)
at com.qa.util.TestUtil.getValueByjpath(TestUtil.java:14)
at com.qa.tests.DataExchangeAdmin.searchreason(DataExchangeAdmin.java:1363)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- searchreason -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="searchreasondetail" time="3.112"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="searchreasonsusers" time="3.059"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="individualsearch" time="3.186"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="entitysearch" time="3.224"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="monthlyentity" time="3.159"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="monthlyindividual" time="3.1"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="downloadreport" time="3.08"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="total_workflow" time="0.895"/>
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="workflow_by_id" time="0.814">
    <failure type="java.lang.AssertionError" message="status code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: status code is not 200 expected [200] but found [400]
at com.qa.tests.DataExchangeAdmin.workflow_by_id(DataExchangeAdmin.java:1549)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- workflow_by_id -->
  <testcase classname="com.qa.tests.DataExchangeAdmin" name="workflow_client" time="0.86"/>
</testsuite> <!-- DataExchangeAdmin -->
